scripts/actions/send-and-update-translation-queue.js

```diff
@@ -0,0 +1,200 @@
+  form.append('localeIdsToAuthorize[]', LOCALE_IDS[locale]);
+  form.append('fileUri', page.file);
+  form.append('file', fs.createReadStream(filepath));
+
+  const url = new URL(
+    `/job-batches-api/v2/projects/${PROJECT_ID}/batches/${batchUid}/file`,
+    process.env.TRANSLATION_VENDOR_API_URL
+  );
+
+  const options = {
+    method: 'POST',
+    headers: {
+      Authorization: `Bearer ${accessToken}`,
+    },
+    body: form,
+  };
+
+  try {
+    const resp = await fetch(url.href, options);
+    const { response } = await resp.json();
+    const { code } = response;
+
+    if (code !== 'ACCEPTED') {
+      console.dir(response, { depth: null });
+      throw new Error(code);
+    }
+
+    console.log(`[*] Successfully uploaded ${page.file}.`);
+    return code;
+  } catch (error) {
+    console.error(`[!] Unable to upload ${page.file}.`);
+    console.error(error);
+    process.exit(1);
+  }
+};
+
+/**
+ * Sends HTML content to the vendor by creating jobs, batches, and uploading
+ * files. On success, this will return the jobUid and batchUid for each locale.
+ * @param {Object<string, Page[]>} content
+ * @returns {Promise<{ jobUids: string[], batchUids: string[]}>}
+ */
+const sendContentToVendor = async (content) => {
+  // 1) Create a job for each locale - save the jobUid for storage
+  const jobRequests = Object.keys(content).map((locale) => {
+    const body = {
+      jobName: `Gatsby Translation Queue (${locale}) ${new Date().toLocaleString()}`,
+      targetLocaleIds: [LOCALE_IDS[locale]],
+    };
+    return vendorRequest(
+      'POST',
+      `/jobs-api/v3/projects/${PROJECT_ID}/jobs`,
+      body
+    );
+  });
+
+  const jobsResponses = await Promise.all(jobRequests);
+  const jobUids = jobsResponses.map((resp) => resp.translationJobUid);
+  console.log(`[*] Successfully created jobs: ${jobUids.join(', ')}`);
+
+  // 2) Create a batch for each job - save bachUid for storage
+  const pages = Object.values(content);
+  const batchRequests = jobUids.map((jobUid, idx) => {
+    const body = {
+      authorize: false,
+      translationJobUid: jobUid,
+      fileUris: pages[idx].map(({ file }) => file),
+    };
+
+    return vendorRequest(
+      'POST',
+      `/job-batches-api/v2/projects/${PROJECT_ID}/batches`,
+      body
+    );
+  });
+
+  const batchResponses = await Promise.all(batchRequests);
+  const batchUids = batchResponses.map((resp) => resp.batchUid);
+  console.log(`[*] Successfully created batches: ${batchUids.join(', ')}`);
+
+  // 3) Upload files to the batches job
+  const accessToken = await getAccessToken();
+  const fileRequests = batchUids.flatMap((batchUid, idx) => {
+    const [locale, localePages] = Object.entries(content)[idx];
+    return localePages.map(uploadFile(locale, batchUid, accessToken));
+  });
+
+  const fileResponses = await Promise.all(fileRequests);
+
+  console.log(`[*] Successfully uploaded ${fileResponses.length} files`);
+
+  return { jobUids, batchUids };
```

Comment:
It doesn't look like you [do anything](https://github.com/newrelic/docs-website/pull/581/files#diff-3275c521907ff8a8e4237b2b80241b357a478e451ec141a3c21300fdfcbb7df0R184) with `jobUids`. Do you need it?

Reply:
Not at the moment, but we might need it for future actions. I _think_ we can just rely on the `batchUid` to check the status of the job. I'll remove this value from the return until we need it.